.github/workflows/e2e.yaml

```diff
@@ -6,522 +6,35 @@ env:
   GO_VERSION: 1.18
 
 jobs:
-  build-operator-image:
-    runs-on: ubuntu-latest
-
-    name: Build the KMMO container image
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Build the image
-        run: docker build -t kmm:local .
-
-      - name: Export the image
-        run: docker save -o kmm_local.tar kmm:local
-
-      - name: Upload the image
-        uses: actions/upload-artifact@v3
-        with:
-          name: ci-images
-          if-no-files-found: error
-          path: kmm_local.tar
-          retention-days: 1
 
   e2e:
     runs-on: ubuntu-latest
 
     name: Prebuilt kernel module
 
-    needs: [build-operator-image]
-
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
 
+      - uses: actions/checkout@v3
       - name: Create the minikube cluster
```

Comment:
```suggestion
      - name: Create the test's infra
```

Reply:
Why change that and give less information?

Reply:
Because: 
* we are not only creating the cluster, we are also building the images.
* minikube is just an implementation BE and may change